docs/guides/allocation-limits.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Set instance allocation limits
+description: Set, view, and change how much time an instance is allowed to use.
+platform: cloud
+---
+# Set instance allocation limits
```

Comment:
Allocation is mandatory but you can decide if the allocation will be limited or if you allow the instance to overconsume (so it being a limit is optional).